src/main.rs

```diff
@@ -14,7 +14,7 @@ const DEFAULT_SKIP_DIR_NAMES: [&str; 3] = [".git", ".rustup", ".cargo"];
 
 fn main() -> Result<()> {
     let mut args = args();
-    if let Some("clean-recursive") = std::env::args().skip(1).next().as_deref() {
+    if let Some("clean-recursive") = std::env::args().nth(1).as_deref() {
```

Comment:
As a general rule, please separate refactoring into separate commits.
(This code has been fixed in master, so please delete this from this PR.)